web/src/app/shared/components/sort/sort.component.ts

```diff
@@ -9,7 +9,7 @@ import { Direction, Sorting } from 'app/shared/models/sorting';
 export class SortComponent implements OnInit {
   @Input() sortingFields: Map<string, string>;
   @Input() menuPosition = 'right';
-  @Input() defaultSortBy = 'key';
+  @Input() defaultSortBy = 'access-id';
```

Comment:
We have to consider that different components are using the sort component. The other components (workbasket-list-toolbar and task-list-toolbar) pass the defaultSortBy via the @ Input. I prefer adding the input in the access-items-management.component.html: <taskana-shared-sort [sortingFields]="sortingFields" (performSorting)="sorting($event)" **[defaultSortBy]='"access-id"'** instead of changing the default value in the sortComponent itself.